backend/typescript/services/implementations/reviewService.ts

```diff
@@ -399,6 +480,266 @@ class ReviewService implements IReviewService {
 
     return result;
   }
+
+  async updateReviews(id: string, entity: ReviewRequestDTO): Promise<void> {
+    try {
```

Comment:
Also couldn't update the featured field (I tried changing it from the default (true) to false and it didn't reflect on the db)

Reply:
fixed